.vscode/launch.json

```diff
@@ -14,7 +14,8 @@
         "sourceMaps": true,
         "env": {
             "NODE_ENV": "development",
-            "DEBUG": "nuxt:*"
+            "DEBUG": "nuxt:*,koa:*",
```

Comment:
I wished we could have more logging and be able to configure through this here.
So far, it seems only `console.log` works, but then it's hard to manage in code by (un|)commenting.